config/config.txt

```diff
@@ -243,7 +243,7 @@ SERVER  byond://golden.beestation13.com:7777
 FORUMURL https://nsv.beestation13.com/phpBB3
 
 ## Wiki address
-WIKIURL https://nsv.beestation13.com/wiki/Main_Page
+WIKIURL https://nsv.beestation13.com/wiki
```

Comment:
Reason why this was changed is because our wiki pages urls are /wiki/____ instead of wiki/Main_Page/____
Changing our wiki urls has really fricked up how books works.

Reply:
Configs tend to need to be changed directly on the hostbox, rather then PRs.

Reply:
Got cha, will remove this file change shortly.

Reply:
Should not cause a change for now. 
Files changed now only displays a one.